charts/system/intel-gpu-plugin/values.yaml

```diff
@@ -0,0 +1,53 @@
+image:
+  repository: intel/intel-gpu-plugin
+  pullPolicy: IfNotPresent
+  tag: 0.30.0@sha256:e96c9955e5aced52f215f7eff77f679c4c400a16a42b1f2c0e8ed9ba76cae3c4
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          args:
+            - -shared-dev-num
+            - "1"
+          probes:
+            liveness:
+              enabled: false
+            readiness:
+              enabled: false
+            startup:
+              enabled: false
+
+portal:
+  open:
+    enabled: false
+
+operator:
+  register: true
+
+persistence:
+  devfs:
+    enabled: true
+    type: device
+    mountPath: /dev/dri
+    hostPath: /dev/dri
+    readOnly: true
+  sysfs:
+    enabled: true
+    type: hostPath
+    mountPath: /sys/class/drm
+    hostPath: /sys/class/drm
+    readOnly: true
+  kubeletsockets:
+    enabled: true
+    type: hostPath
+    mountPath: /var/lib/kubelet/device-plugins
+    hostPath: /var/lib/kubelet/device-plugins
```

Comment:
```suggestion
    hostPath: /var/lib/kubelet/device-plugins

```

Reply:
Did not know an extra line was needed at the end.